spec/lib/services/get_an_identity_service/webhooks/user_updated_processor_spec.rb

```diff
@@ -38,6 +38,26 @@
     let(:new_trn) { rand(1_000_000..9_999_999).to_s }
     let(:new_trn_status) { "found" }
 
+    it "updates user data and sends it to ECF" do
```

Comment:
I think this should go as well?

Reply:
I think we just change the name, it doesn't send to ECF, only updates